packages/dappmanager/src/initializeDb.ts

```diff
@@ -71,7 +72,9 @@ export default async function initializeDb(): Promise<void> {
   //   UPnP is available and necessary only if the internalIp is not equal to the public IP
   //   and the external IP from UPnP command succeeded
   const upnpAvailable = publicIp
-    ? Boolean(externalIp && internalIp !== publicIp)
+    ? Boolean(externalIp && internalIp !== publicIp) && (await upnpc.list())
```

Comment:
Doesn't upnpc.list() throw if UPnP is not available?

Reply:
yes, and aren't this exactly what we want to set upnpavailable to false (if the function throws an error)

Reply:
Try it out, list() throwing will interrupt the entire initializeDb function and nothing will be set

Reply:
Then this error is a "controlled error" since we know the output and the meaning: upnp device is not available

In the file https://github.com/dappnode/DNP_DAPPMANAGER/blob/master/packages/dappmanager/src/modules/upnpc/parseGeneralErrors.ts

Should we change
```
if (noUpnpAvailable) throw Error("NOUPNP: No UPnP device available");
```

to 

```
if (noUpnpAvailable) {
   db.upnpavailable.set(true)
}
```

Or something similar

Reply:
upnp errors typed on last commit